CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Aptos Python SDK will be captured in this file. This changelog is written by hand for now.
 
+## Unreleased
+- Set max Uleb128 to MAX_U32
+- Add Behave behavioral specifications for BCS and AccountAddress
```

Comment:
maybe instead of features we should call it validator or validation? and maybe add something to our readme or contributors doc?

Reply:
The folder?  It's required I think for the behavioral spec framework